src/Cmdlets/GetKubeResourceCmdlet.cs

```diff
@@ -11,22 +11,52 @@ namespace Kubectl.Cmdlets {
     [Cmdlet(VerbsCommon.Get, "KubeResource")]
     [OutputType(new[] { typeof(KubeResourceV1) })]
     public sealed class GetKubeResourceCmdlet : KubeApiCmdlet {
```

Comment:
As a note, I planned to obsolete the specific `Get-KubePod`,`Get-KubeDeployment` etc in favor of just this cmdlet

Reply:
Good to know. I can revert the changes to those files. I believe it would be a good idea to push a warning message about the deprecation to avoid someone implementing features against this cmdlets.